### DISCUSSION TITLE
https://thanhle.blog/blog/nimbus-an-extension-to-turn-boring-address-into-usefull-info

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#14] [Ideas]
# https://thanhle.blog/blog/nimbus-an-extension-to-turn-boring-address-into-usefull-info

https://thanhle.blog/blog/nimbus-an-extension-to-turn-boring-address-into-usefull-info

**Comment by vinhloc30796** [comment]
Awesome anh! Looks very polished already for a v1.0

**Reply by thanhlmm** [reply]
Thank you! I'll let you know when it release
